core/domain/topic_domain.py

```diff
@@ -421,6 +431,31 @@ def __init__(
         self.version = version
         self.story_reference_schema_version = story_reference_schema_version
 
+    @classmethod
+    def from_dict(cls, topic_dict):
```

Comment:
Oh, sorry, there was no need to add this function. I thought this function already existed and just wanted to check if this was updated. Since it didn't exist before, and if you are not using this anywhere else, you can remove it.

Reply:
Done.